test/test.js

```diff
@@ -256,4 +260,11 @@ describe('HTTP Request testing', () => {
         assert.equal(res.tasks[0].id, '3VtEMGudjwjjM60j7deSI987');
       });
   });
+  it('Get compliance information from tasks assigned to Onfleet drivers', () => {
```

Comment:
I would also try including one other unit test in here to confirm that the `turnByTurn` object isn't empty when a google API key is present. Not required though, would just be a good test to include if it's not a ton of work

Reply:
Done